bin/vcf_reader.py

```diff
@@ -0,0 +1,1180 @@
+import os
+import sys
+import logging
+import csv
+import re
+import vcf
+import argparse
+import urllib2
+import itertools
+import pandas as pd
+import numpy as np
+import Fred2.Core.Generator as generator
+import math
+
+from collections import defaultdict
+from Fred2.IO.MartsAdapter import MartsAdapter
+from Fred2.Core.Variant import Variant, VariationType, MutationSyntax
+from Fred2.EpitopePrediction import EpitopePredictorFactory
+from Fred2.IO.ADBAdapter import EIdentifierTypes
+from Fred2.IO.UniProtAdapter import UniProtDB
+from Fred2.Core.Allele import Allele
+from Fred2.Core.Peptide import Peptide
+from Fred2.IO import FileReader
+from Bio import SeqUtils
+
+from datetime import datetime
+from string import Template
+
+__author__ = 'mohr, walzer'
+VERSION = "1.1"
+
+ID_SYSTEM_USED = EIdentifierTypes.ENSEMBL
+transcriptProteinMap = {}
+transcriptSwissProtMap = {}
+
+# Adapt depending on system
+REF_PATH = ['/lustre_cfc/qbic/reference_genomes/IRMA_references/uniprot_all_swissprot_UP000005640.fasta', '/lustre_cfc/qbic/reference_genomes/IRMA_references/Homo_sapiens.GRCh38.pep.all.fa']
```

Comment:
How is this supposed to work?
People are not supposed to edit files.

Reply:
oh right, yes that we should take out